Xidicone.AgentSVN version 2.73

jussij opened the pull request:
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----


###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/43327)

jussij committed:
Xidicone.AgentSVN version 2.73

ItzLevvie commented on manifests/x/Xidicone/AgentSVN/2.73/Xidicone.AgentSVN.installer.yaml:
```diff
@@ -0,0 +1,21 @@
+
+PackageIdentifier: Xidicone.AgentSVN
+PackageVersion: 2.73
+InstallerSwitches:
+  Silent: /verysilent
+  SilentWithProgress: /silent
```
```suggestion
```

ItzLevvie replied:
This isn't needed because it's the default for `InstallerType: inno`

jussij committed:
Update manifests/x/Xidicone/AgentSVN/2.73/Xidicone.AgentSVN.installer.yaml

Co-authored-by: Levvie - she/her <11600822+ItzLevvie@users.noreply.github.com>

jussij commented:
Is there something wrong with this pull request?

I only ask because it appears to have sitting in a blocked state for several hours.

I'm planning to update a further two manifests with their latest software version information, but I'm holding of doing this until I know I've done this first update correctly.

ryfu-msft commented:
@wingetbot waivers Add Validation-Domain

ryfu-msft commented:
@jussij, sorry for the delay, I had overlooked reviewing this PR. Once the build passes, you should be good to go :)

jussij committed:
Update Xidicone.AgentSVN.installer.yaml

Updated file with new Sha256 details and also update file to match improvement suggestions from earlier manifest submissions.

jussij committed:
Update Xidicone.AgentSVN.locale.en-US.yaml

Update file with improvement suggestions from earlier change request.
